core-bundle/src/Twig/Interop/FragmentTemplate.php

```diff
@@ -0,0 +1,700 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Twig\Interop;
+
+use Contao\CoreBundle\Asset\ContaoContext;
+use Contao\FilesModel;
+use Contao\Model;
+use Contao\Template;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * This class is a simple container object for template data.
+ */
+final class FragmentTemplate /* TODO: remove base class in Contao 6 */ extends Template
+{
+    /**
+     * @var array<string,mixed>
+     */
+    private array $context = [];
+
+    /**
+     * @param \Closure(self):Response $onGetResponse
+     *
+     * @internal
+     * @noinspection MagicMethodsValidityInspection
+     */
+    public function __construct(private string $templateName, private \Closure $onGetResponse)
+    {
```

Comment:
```suggestion
    {
        // Do not call parent constructor
```

AFAIK we add a comment if not calling the parent constructor is on purpose.